ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -433,7 +433,7 @@ public struct Avatar: View, TokenizedControlView {
             var hash: Int32 = 0
             for len in (0..<text.length).reversed() {
                 let ch = text.character(at: len)
-                let shift = len % 8
+                let shift = Int32(len % 8)
```

Comment:
Would it be useful to add a comment on why converting to Int32 is necessary?

Reply:
Sure! This function is a little opaque in general.

Reply:
Comment added. Also this whole PR just reminds me how badly I want to refactor this codebase to allow for code sharing between iOS and macOS.